blog/utils/records.ts

```diff
@@ -11,5 +11,11 @@ export async function getRecordsByPath<T>(
   const current = Object.entries(resolvables).flatMap(([key, value]) => {
     return key.startsWith(path) ? value : [];
   });
-  return (current as Record<string, T>[]).map((item) => item[accessor]);
+  return (current as Record<string, T>[]).map((item) => {
+    const id = (item.name as string).split(path)[1]?.replace("/", "");
+    return {
+      ...item[accessor],
+      id,
+    };
+  });
```

Comment:
why add it manually? I think this should be in the table, maybe by a workflow that runs it when this changes comes to production

Reply:
> why add it manually? I think this should be in the table, maybe by a workflow that runs it when this changes comes to production

@IncognitaDev, by default it already returns, but in a different format.

ex: `collections/blog/categories/05430797e2d1` or `collections/blog/posts/1d76260ef8a8`

the objective is to do the split and return only the unique identifier to use later

Reply:
example of how I'm using

![{A6300D91-C6DA-4B99-BFC2-5B4070847CD3}](https://github.com/user-attachments/assets/d56c3b04-032b-409d-8103-cc95eadc89fa)